lib/oli/utils/slug.ex

```diff
@@ -172,12 +172,23 @@ defmodule Oli.Utils.Slug do
   def slugify(nil), do: ""
 
   def slugify(title) do
-    String.downcase(title, :default)
+    case String.downcase(title, :default)
     |> String.trim()
     |> String.replace(" ", "_")
     |> alpha_numeric_only()
     |> URI.encode_www_form()
-    |> String.slice(0, 30)
+    |> String.slice(0, 30) do
```

Comment:
Nit but I think this long case clause could be more readable by piping the result into a new function or even piping it into the case like `|> case do`